openapi.yaml

```diff
@@ -17139,6 +17145,25 @@ components:
             enough for it to be considered unhealthy and taken out of rotation.
           example: true
           x-linode-cli-display: 6
+        proxy_protocol:
+          description: >
+            ProxyProtocol is an internet protocol that sends information as a header on a
+            TCP connection without losing client data. When used with a NodeBalancer, it
+            enables the termination of TCP connections.
+
+
+            Linode NodeBalancers supports two versions, `v1` and `v2`. `v1` is human readable
```

Comment:
Another comment. Comparing this to other descriptions in the same section (e.g. `stickiness`) , this is better be formatted as

 *  If set to `none` NodeBalancer doesn't send any auxiliary data over TCP connections
 *  If set to `v1` the version 1 (human readable) of ProxyProtocol is used
 *  If set to `v2` the version 2 (binary) of ProxyProtocol is used